Procfile

```diff
@@ -1,4 +1,4 @@
-web: make clean && python2.7 manage.py runserver 0.0.0.0:8000
+web: make clean && gunicorn kuma.vagrant-wsgi -b 0.0.0.0:8000
```

Comment:
That means we'll stop using the django dev server which serves static files using the staticfiles system and instead uses whitenoise in development. I wonder if that means we'll have to run collectstatic before we run it now. Can you double check if a changed file shows up on the site without restarting Foreman?